app/Http/Controllers/Admin/ModulesController.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Contracts\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Str;
+use Laracasts\Flash\Flash;
+use Madnest\Madzipper\Madzipper;
+use App\Models\Module;
+
+class ModulesController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @param Request $request
+     *
+     * @return mixed
+     */
+    public function index()
+    {
+        $modules = Module::all();
+        return view('admin.modulesmanager.index', [
+            'modules' => $modules
+        ]);
+    }
+
+    public function addModule(Request $request)
```

Comment:
Sorry for being nitpicky - I just want all the code to be uniform. Rename all these methods to the same as in other controllers - they're the standard names Laravel uses, so `index()`, `update()`, etc.

Reply:
Okay, shall I use add() or create() for adding module GET/POST?
I have used store() for storing module

Reply:
Follow the link I sent from the Laravel docs for those conventions